### ISSUE TITLE
Plugin auto install for IE

### ISSUE THREAD

**Issue by jauhenk** [Temasys/AdapterJS#82]
Hey guys! Is it any possibility to run auto installation of the plugin for IE?

**Comment by johache** [comment]
Can you be more specific about what you mean by auto-installation?
If you mean start the download immediately on arriving on the page, sure, you can do that. Just access the download link defined in the adapter:
AdapterJS.WebRTCPlugin.pluginInfo.downloadLink

**Comment by jauhenk** [comment]
Sure: I mean can we run immediately plugin installation with ActiveX?
http://stackoverflow.com/questions/6122437/firebreath-plugin-automatic-installation

**Comment by johache** [comment]
Hi,
That is SUPER interesting. However, reading pmont's answer, it seems like the binaries have to be signed.
We currently do not sign the free plugin. We do support plugin signing for our contracted customer though. I this case I think you might be able to modify the injection of the plugin done by the adapter to use this feature.
To be tested.

**Comment by soulhunter1987** [comment]
Yes, that would be a super feature. Most users with lack of WebRTC support are on IE, and most of them are afraid of manually installing additional software, till most of it is malware in some way (they do not know what they a really installing - a browser plugin or a virus) Auto-installation in terms of a browser will be really helpfull here.

**Comment by johache** [comment]
Agreed.
@agouaillard  @ncurrier @serrynaimo 
What do you think? Is it worth getting a certificate and starting to sign all free plugins? What would be the cost?

**Comment by agouaillard** [comment]
take that discussion offline with thomas and nath.

On Thu, May 21, 2015 at 10:07 AM, Jacques-Olivier Haché <
notifications@github.com> wrote:

> Agreed.
> @agouaillard <https://github.com/agouaillard> @ncurrier
> <https://github.com/ncurrier> @serrynaimo <https://github.com/serrynaimo>
> What do you think? Is it worth getting a certificate and starting to sign
> all free plugins? What would be the cost?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Temasys/AdapterJS/issues/82#issuecomment-104188236>.
>



-- 
Alex. Gouaillard, PhD, PhD, MBA
------------------------------------------------------------------------------------
CTO - Temasys Communications, S'pore / Mountain View
President - CoSMo Software, Cambridge, MA
------------------------------------------------------------------------------------
sg.linkedin.com/agouaillard

   -

**Comment by contra** [comment]
Any update on this?

**Comment by serrynaimo** [comment]
It's on the list for the free plugin and already available on the commercial side.

**Comment by soulhunter1987** [comment]
Any release plan for this? Or currently not planed?
